.github/workflows/pr.yaml

```diff
@@ -105,5 +105,5 @@ jobs:
       cuda: '["12.2"]'
       build_command: |
         sccache -z;
-        build-all -DBUILD_PRIMS_BENCH=ON -DBUILD_ANN_BENCH=ON --verbose;
+        build-all --verbose;
```

Comment:
Oh- good find. I guess we don't need this in cuVS, do we?

Reply:
Yeah, it was a good catch - @trxcllnt fixed this in https://github.com/rapidsai/cuvs/pull/75/commits/a6e1436a010e672230b1511d754d81a918b2a703